src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const toObject = {};
+
+  sourceString.split(';').forEach((item) => {
+    const [key, value] = item.split(':');
+
+    if (key && value) {
+      toObject[key.trim()] = value.trim();
+    }
```

Comment:
The current implementation is correct, but it's important to note that it does not handle cases with multiple colons within the property values or multiple semicolons without any styles between them. While it's not required for this task, handling these edge cases would make the function more robust.